test/index.html

```diff
@@ -5,11 +5,12 @@
 <link rel="stylesheet" href="../build/test.css">
 <style>
   .fixture-Box {
-    background: rgb(100, 150, 50);
-    background: rgb(100, 150, 50);
-    border: 1px solid black;
+    background: rgba(100, 150, 50, .75);
```

Comment:
The main reason was to copy the nesting examples here http://philipwalton.github.io/solved-by-flexbox/demos/grids/ but I changed my mind. I can revert this.